app/packages/app/src/components/ImageContainerHeader.tsx

```diff
@@ -54,35 +60,32 @@ const SliderContainer = styled.div`
   padding-right: 1rem;
 `;
 
-const Count = () => {
-  const element = useRecoilValue(fos.elementNames);
-  const total = useRecoilValue(
-    fos.count({ path: "", extended: false, modal: false })
-  );
-
-  return (
-    <RightDiv>
-      <div>
-        <PathEntryCounts modal={false} path={""} />
-        &nbsp;
-        {total === 1 ? element.singular : element.plural}
-      </div>
-    </RightDiv>
-  );
-};
-
 const ImageContainerHeader = () => {
+  const type = useRecoilValue(fos.dataset).mediaType;
   const setGridZoom = useSetRecoilState(gridZoom);
   const gridZoomRangeValue = useRecoilValue(gridZoomRange);
   const theme = useTheme();
   const group = useRecoilValue(isGroup);
+  const element = useRecoilValue(fos.elementNames);
+  const total = useRecoilValue(
+    fos.count({ path: "", extended: false, modal: false })
+  );
 
   return (
     <SamplesHeader>
       <GridActionsRow />
       <RightContainer>
         <Suspense fallback={<RightDiv>{"Loading..."}</RightDiv>}>
-          <Count />
+          <PillButton
+            icon={<MediaTypeIcon type={type} />}
+            text={<PathEntryCounts modal={false} path={""} />}
+            title={`${total.toLocaleString()} ${_.capitalize(type)} ${
```

Comment:
I don't think the type needs to be capitalized. The values are not capitalized in Python, e.g.:

```
Name:        quickstart
Media type:  image
Num samples: 200
Persistent:  False
Tags:        []
Sample fields:
    id:           fiftyone.core.fields.ObjectIdField
    filepath:     fiftyone.core.fields.StringField
    tags:         fiftyone.core.fields.ListField(fiftyone.core.fields.StringField)
    metadata:     fiftyone.core.fields.EmbeddedDocumentField(fiftyone.core.metadata.ImageMetadata)
    ground_truth: fiftyone.core.fields.EmbeddedDocumentField(fiftyone.core.labels.Detections)
    uniqueness:   fiftyone.core.fields.FloatField
    predictions:  fiftyone.core.fields.EmbeddedDocumentField(fiftyone.core.labels.Detections)
```